### ISSUE TITLE
Website discontinued 

### ISSUE THREAD

**Issue by Agosh13** [covidpass-org/covidpass#177]
Website is not available anymore but it’s still needed for people that have received the green pass after the covid-19 (functionality not available on iOS 15.4) pathology.

**Comment by Ein-Tim** [comment]
@Agosh13 

You are right that EU digital certificates of recovery can't be added to the Wallet via the iOS health app.

**Comment by SkYNewZ** [comment]
Indeed, we can add our certificate in case of a positive result, but this is considered as a pathology and not as a vaccination. We can't see the QR code in Wallet... Thanks Apple

**Comment by marvinsxtr** [comment]
I would recommend using https://covidpassi.fi instead, although it's in Finnish it should be usable.
